### PR TITLE
What's new for preview 3

### LINKED ISSUE #10589: What's new for .NET Core 3.0 Preview 3
Document the new info about Core 3.0 Preview 3

**File in the repo**  
docs\core\whats-new\dotnet-core-3-0.md

### PR DESCRIPTION
**Pull request by Thraka**
Updated info for what's new based on the blog post.

Fixes #10589
Replaces #11797

@mairaw I rewrote, cut, restructured everything. I may have some things re-added at a later time. However, preview 4 is out now, so more updates coming. Might as well get this published though.

* [x] Have to create the PR for the code changes.

### CODE REVIEW

**Comment by Thraka** [comment]
@mairaw Can you review my PR please ? 😄

File: docs/core/whats-new/dotnet-core-3-0.md [hunk 1]
```diff
@@ -262,534 +157,182 @@ dotnet new wpf
...
-  </PropertyGroup>
-</Project>
-```
+For more information about how to port an existing .NET Framework application, see [Port WPF projects](../porting/winforms.md) and [Port Windows Forms projects](../porting/wpf.md).
 
-Both `UseWPF` and `UseWindowsForms` can be set to `true` if the app uses both frameworks, for example when a Windows Forms dialog is hosting a WPF control.
 
-Please share your feedback on the [dotnet/winforms](https://github.com/dotnet/winforms/issues),  [dotnet/wpf](https://github.com/dotnet/wpf/issues) and [dotnet/core](https://github.com/dotnet/core/issues) repos.
 
 ## MSIX Deployment for Windows Desktop
```
Comment (BillWagner) [inline docs/core/whats-new/dotnet-core-3-0.md]: Should we define MSIX on its first use?

File: docs/core/whats-new/dotnet-core-3-0.md [hunk 2]
```diff
@@ -262,534 +157,182 @@ dotnet new wpf
...
-}
-```
 
-## Type: MetadataLoadContext
 
-The `MetadataLoadContext` type has been added that enables reading assembly metadata without affecting the caller’s application domain. Assemblies are read as data, including assemblies built for different architectures and platforms than the current runtime environment. `MetadataLoadContext` overlaps with the <xref:System.Reflection.Assembly.ReflectionOnlyLoad*>, which is only available in the .NET Framework.
+## Interop improvements
 
-`MetdataLoadContext` is available in the [System.Reflection.MetadataLoadContext package](https://www.nuget.org/packages/System.Reflection.MetadataLoadContext). It is a .NET Standard 2.0 package.
+Enhancements have been added to .NET Core that improves native API interop.
```
Comment (BillWagner) [inline docs/core/whats-new/dotnet-core-3-0.md]:
```suggestion
.NET Core 3,0 improves native API interop.
```

File: docs/core/whats-new/dotnet-core-3-0.md [hunk 3]
```diff
@@ -262,534 +157,182 @@ dotnet new wpf
...
-byte[] nonce = new byte[12];
-RandomNumberGenerator.Fill(nonce);
+## Cryptography ciphers
 
-// normally this would be your data
-byte[] dataToEncrypt = new byte[1234];
-byte[] associatedData = new byte[333];
-RandomNumberGenerator.Fill(dataToEncrypt);
-RandomNumberGenerator.Fill(associatedData);
+Support has been added for **AES-GCM** and **AES-CCM** ciphers, implemented with <xref:System.Security.Cryptography.AesGcm?displayProperty=nameWithType> and <xref:System.Security.Cryptography.AesCcm?displayProperty=nameWithType> respectively. These algorithms are both [Authenticated Encryption with Association Data (AEAD) algorithms](https://en.wikipedia.org/wiki/Authenticated_encryption).
```
Comment (BillWagner) [inline docs/core/whats-new/dotnet-core-3-0.md]:
```suggestion
.NET 3.0 add support for **AES-GCM** and **AES-CCM** ciphers, implemented with <xref:System.Security.Cryptography.AesGcm?displayProperty=nameWithType> and <xref:System.Security.Cryptography.AesCcm?displayProperty=nameWithType> respectively. These algorithms are both [Authenticated Encryption with Association Data (AEAD) algorithms](https://en.wikipedia.org/wiki/Authenticated_encryption).
```

File: docs/core/whats-new/dotnet-core-3-0.md [hunk 4]
```diff
@@ -262,534 +157,182 @@ dotnet new wpf
...
-* Alpine: 3.8+
-
-Chip support follows:
+## ARM64 Linux support
 
-* x64 on Windows, macOS, and Linux
-* x86 on Windows
-* ARM32 on Windows and Linux
-* ARM64 on Linux
+Support has been added for ARM64 for Linux. The primary use case for ARM64 is currently with IoT scenarios. For more information, see [.NET Core ARM64 Status](https://github.com/dotnet/announcements/issues/82).
```
Comment (BillWagner) [inline docs/core/whats-new/dotnet-core-3-0.md]:
```suggestion
.NET Core 4.0 adds support for ARM64 for Linux. The primary use case for ARM64 is currently with IoT scenarios. For more information, see [.NET Core ARM64 Status](https://github.com/dotnet/announcements/issues/82).
```

**Review by BillWagner — APPROVED** [review]
Overall, this is really good @Thraka 

I had a few suggestions here and there that you can consider.  You can :shipit: when you are ready.

File: docs/core/whats-new/dotnet-core-3-0.md [hunk 5]
```diff
@@ -6,252 +6,147 @@ dev_langs:
...
+There are some operations, like linking and razor page publishing that will still require publishing.
 
-When the local tools manifest file is available, but the tools defined in the manifest have not been installed, use the following command to automatically download and install those tools:
 
-```console
-dotnet tool restore
-```
 
-Run a local tool with the following command:
+## Local dotnet tools
```
[the remark this answers is not in the document]
Comment (Thraka) [reply docs/core/whats-new/dotnet-core-3-0.md]: I'll just simplify it completely.

```suggestion
## Local tools
```

File: docs/core/whats-new/dotnet-core-3-0.md (hunk 1 above)
Comment (Thraka) [reply docs/core/whats-new/dotnet-core-3-0.md]: Going to the link for this, they don't define it either. it's just called MSIX 🤣

**Pull request merged**